### PR TITLE
Enhance/subblock to graph

### PR DESCRIPTION
**Pull request by Superjomn**

### CODE REVIEW

**Comment by sfraczek** [comment]
I have merged this code to develop and I see it draws the while contents.
What is the scope of work done/ToBeDone here?
 Is this PR supposed to fix the passes applying only to block 0 or is it supposed to enable drawing graph only?

File: paddle/fluid/framework/ir/infer_clean_graph_pass.cc [hunk 1]
```diff
@@ -12,56 +12,10 @@
...
-
-    auto is_valid_node = [](Node* x) {
-      return x && IsControlDepVar(*x) && x->IsVar() && !x->Var();
-    };
-
-    std::unordered_set<const Node*> invalid_nodes;
-    int valid_op = 0;
-    for (auto* node : graph->Nodes()) {
-      PADDLE_ENFORCE_NOT_NULL(node);
-      if (is_valid_node(node)) {
```
Comment (sfraczek) [inline paddle/fluid/framework/ir/infer_clean_graph_pass.cc:41]: Did you mean to call this function `is_invalid_node`?

File: paddle/fluid/framework/ir/graph_viz_pass.cc [hunk 2]
```diff
@@ -82,14 +95,53 @@ std::unique_ptr<ir::Graph> GraphVizPass::ApplyImpl(
       {Dot::Attr("style", "filled,rounded"), Dot::Attr("shape", "box"),
        Dot::Attr("fillcolor", "yellow")});
 
-  auto marked_nodes = ConsumeMarkedNodes(graph.get());
+  const std::vector<Dot::Attr> sub_graph_attrs(
+      {Dot::Attr("style", "filled,rounded,dotted"),
+       Dot::Attr("fillcolor", "black"), Dot::Attr("fillcolor", "azure"),
```
Comment (sfraczek) [inline paddle/fluid/framework/ir/graph_viz_pass.cc]: Does azure override black here?

**Comment by Superjomn** [comment]
This PR just support drawing graph.

**Pull request closed without merging**